otter/assign/cell_factory.py

```diff
@@ -132,7 +132,8 @@ def create_export_cells(self):
             args += ["force_save=True"]
         if self.assignment.export_cell.run_tests:
             args += ["run_tests=True"]
-
+        if len(self.assignment.export_cell.files) != 0:
+            args += [f"files={self.assignment.export_cell.files}"] # TODO: add the arguments
```

Comment:
Please imeplement this TODO in this PR or add more details on what work is remaining to the comment.